lib/utils.js

```diff
@@ -56,8 +56,16 @@ const buildUrl = (host, port, ssl, endpoint) => {
 };
 
 // Custom Logger
-const log = (msg) => {
-    console.log('[' + new Date().toISOString().substring(2, 19).replace('T', ' ').replace(/[Z-]/g, '') + '] ' + msg);
+const log = (message, level) => {
+    const dateObject = new Date();
+    const padStart = (number, expectedLength) => {
+        const strNumber = String(Math.abs(number)).padStart(expectedLength, '0');
+        return (number < 0) ? `-${strNumber}` : strNumber;
+    };
+    const date = `${dateObject.getFullYear()}-${padStart(dateObject.getMonth() + 1, 2)}-${padStart(dateObject.getDate(), 2)}`;
+    const time = `${padStart(dateObject.getHours(), 2)}:${padStart(dateObject.getMinutes(), 2)}:${padStart(dateObject.getSeconds(), 2)}`;
+    const timezone = padStart(dateObject.getTimezoneOffset() * -5 / 3, 4);
```

Comment:
Even though the calculation is correct, it's confusing to readers 🤣. How about 
```
dateObject.getTimezoneOffset() * -1  / 60 * 100
```

Reply:
you are right! updated it to be `dateObject.getTimezoneOffset() * -100  / 60`